### PR TITLE
hawkular-parent pom.xml copied from hawkular git repository plus some tweaks

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:git@github.com:hawkular/hawkular.git</connection>
```
Comment (pilhuhn) [inline pom.xml]: Looks odd

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    <url>https://github.com/hawkular</url>
+    <tag>HEAD</tag>
+  </scm>
+  <issueManagement>
+    <system>jira</system>
+    <url>https://issues.jboss.org/browse/HAWKULAR</url>
+  </issueManagement>
+  <ciManagement>
+    <system>hudson</system>
+    <url>https://hudson.jboss.org/hudson/job/hawkular</url>
```
Comment (pilhuhn) [inline pom.xml]: hm? Do / will we have that?

Comment (ppalaga) [inline pom.xml]: Good question. This is how it was in hawkular project. Let me remove it. Makes no sense here.

File: pom.xml (hunk 1 above)
Comment (ppalaga) [inline pom.xml]: Good catch. Going to fix

**Pull request merged**